.circleci/config.yml

```diff
@@ -271,6 +274,12 @@ jobs:
           command: |
             IMG="image-$(date +%j).gz"; docker save addons-frontend | pigz --fast -c > ~/addons-frontend/docker/$IMG; ls -l ~/addons-frontend/docker
             ls -l ~/addons-frontend/docker
+      - run:
+          name: Check image works
+          command: |
+            docker run --rm -p 4000:4000 -d -e NODE_ENV=production -e NODE_CONFIG_ENV=prod addons-frontend
+            curl --retry-all-errors --retry 10 --retry-delay 2 http://127.0.0.1:4000/__frontend_lbheartbeat__
+            docker stop addons-frontend
```

Comment:
if you are adding `--rm`  then why also add docker stop?

Reply:
I'm starting in daemon mode so that it runs in the background. The `--rm` is actually a bad idea now that I've moved things around since we need to re-use that image in later steps! I'll remove it.

Reply:
a32637e5f048c63a62b020739a0cb4e8ec305db6 simplifies this a bit